src/dash-thumbnail-controller.js

```diff
@@ -0,0 +1,107 @@
+// @flow
+import {AdaptationSet} from './parser/adaptation-set';
```

Comment:
I think the prefix dash is redundant you can call it `ThumbnailController` cause you already on the dash adapter

Reply:
This is helpful for debugging and to find files easily (can be more than one thumbnail controller on a page).